azurestack/resource_arm_virtual_machine_scale_set.go

```diff
@@ -622,15 +622,15 @@ func resourceArmVirtualMachineScaleSetCreate(d *schema.ResourceData, meta interf
 	}
 
 	storageProfile := compute.VirtualMachineScaleSetStorageProfile{}
```

Comment:
do we also want to support Managed Disks for VMSS?

Reply:
That'll come in a separate PR